src/v2/Apps/Collect/Routes/Collect/index.tsx

```diff
@@ -54,77 +54,72 @@ export const CollectApp: React.FC<CollectAppProps> = ({
   const tracking = useTracking()
 
   return (
-    <AnalyticsContext.Provider
-      value={{
-        contextPageOwnerType: OwnerType.collect,
-      }}
-    >
-      <AnalyticsContext.Consumer>
-        {({ contextPageOwnerType }) => (
-          <AppContainer>
-            <FrameWithRecentlyViewed>
-              <Title>{title}</Title>
-              <Meta property="og:url" content={`${sd.APP_URL}/collect`} />
-              <Meta
-                property="og:image"
-                content={`${sd.APP_URL}/images/og_image.jpg`}
+    <AnalyticsContext.Consumer>
```

Comment:
For FC components at least, can we replace this renderProps form with the `useAnalyticsContext` hook throughout?